main.go

```diff
@@ -233,6 +267,8 @@ func (opt *Options) bindCLIFlagsToViper(f *pflag.FlagSet) error {
 	f.BoolVar(&opt.EnableToolUseShim, "enable-tool-use-shim", opt.EnableToolUseShim, "enable tool use shim")
 	f.BoolVar(&opt.Quiet, "quiet", opt.Quiet, "run in non-interactive mode, requires a query to be provided as a positional argument")
 
+	f.Var(&opt.UserInterface, "user-interface", "user interface mode to use")
```

Comment:
pl. add supported values "terminal, html, defaults to html" to help with discovery.